site/data/data.json

```diff
@@ -2548,6 +2645,21 @@
       }
     ]
   },
+  {
+    "hide": true,
+    "name": "Black Cat Barbershop",
+    "website": "https://www.blackcatbarbershop.com/",
+    "note": "",
+    "workers": [
+      {
+        "name": "Black Cat Barbershop",
+        "paypal": "blackcatbarbershop",
+        "venmo": "",
+        "cashapp": "",
+        "last4": ""
+      }
+    ]
+  },
```

Comment:
Hid this entry because it is the info for the business rather than any specific worker(s).

Reply:
Yea, this is a grey area. I would probably allow it, and let people make their own decisions.